### PR TITLE
Fix README internal link

### PR DESCRIPTION
**Pull request by spenczar**
Just fixing an internal link reference that has changed on the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ you can use the following in your init.el:
 
 ## Clippy
 `rust-run-clippy` runs
-[Clippy](https://github.com/rust-lang/rust-clippy), a linter. 
+[Clippy](https://github.com/rust-lang/rust-clippy), a linter.
```
Comment (spenczar) [inline README.md:113]: Sorry about this unrelated whitespace change. I have a `'delete-trailing-whitespace` `'before-save-hook` which is responsible.

Comment (mookid) [reply README.md:113]: No worries.

**Comment by mookid** [comment]
Thanks!
